src/routes/ethereum-navigator/+page.server.ts

```
@@ -0,0 +1,113 @@
+        }
+
+        const data: BeehiivResponse = await response.json();
+        
+        const filteredAndSortedPosts = data.data
+            .filter(isValidPost)
+            .sort(comparePosts);
+
+        return { posts: filteredAndSortedPosts };
+        
+    } catch (err) {
+        const handledError = logError('newsletter content fetch', err, {
```

ESLint

[error] 'handledError' is assigned a value but never used.

(@typescript-eslint/no-unused-vars)